src/Tests/Microsoft.DotNet.Cli.Utils.Tests/TypoCorrectionTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#nullable enable
+
+using FluentAssertions;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.Utils.Tests
+{
+    public class TypoCorrectionTests
+    {
+        [InlineData("wbe", "web|webapp|install|uninstall", "web")]
+        [InlineData("uninstal", "web|webapp|install|uninstall", "uninstall")]
```

Comment:
It would be nice to have a testcase validating return of multiple matches in a proper order by edit distance